master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -164,7 +165,23 @@ private[celeborn] class Master(
   }
   masterSource.addGauge(MasterSource.WORKER_COUNT) { () => statusSystem.workers.size }
   masterSource.addGauge(MasterSource.LOST_WORKER_COUNT) { () => statusSystem.lostWorkers.size }
+  masterSource.addGauge(MasterSource.RUNNING_APPLICATION_COUNT) { () =>
+    statusSystem.appHeartbeatTime.size
+  }
   masterSource.addGauge(MasterSource.PARTITION_SIZE) { () => statusSystem.estimatedPartitionSize }
+  masterSource.addGauge(MasterSource.PARTITION_WRITTEN) { () =>
+    userResourceConsumptions.values().stream()
```

Comment:
userResourceConsumptions are updated when `applyAllFallbackPolicy` is called. It's not correct to use it here.